roles/openshift_register_nodes/library/kubernetes_register_node.py

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 # vim: expandtab:tabstop=4:shiftwidth=4
+# pylint: disable=fixme, missing-docstring, too-many-arguments
+# pylint: disable=too-many-locals, too-many-branches, too-few-public-methods
+# pylint: disable=no-self-use
```

Comment:
Do all of these have to be disabled?

For instance, docstrings are extremely easy to create, and don't require much testing (if any). Can you just quickly fix the docstrings?

And are these pylint tests meant to be permanently disabled or temporarily disabled?

Please add comments saying why they're being disabled, and if it's meant to be temporary or permanent.